src/typings-new-format/spec-1.0/common.ts

```diff
@@ -1,28 +1,50 @@
-export interface Annotable {
+export interface SourceInfo{
+
+    /**
+     * Path to file which contains definition
+     */
+    path?: string
+
+    /**
+     * Namespace of defining library if any
+     */
+    namespace?: string
+
+}
+
+export interface HasSource extends SourceInfo{
+
+    scalarsSources?: { [key:string]:SourceInfo[] }
```

Comment:
Not sure what that actually means. No documentation makes it difficult to understand that.